metafix/src/test/resources/org/metafacture/metafix/integration/method/fromJson/toJson/lookupSimpleWithInternalMapDeleteNotMatchingLiteral/expected.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name" : "Jake",
+  "type" : "mammal"
+}
+{
+  "name" : "Blacky",
+  "type" : "bird"
+}
+{
+  "name" : "Jens",
+  "type" : "human"
```

Comment:
This is not expected, is it? "Not matching literal" should be deleted, right? Then this test can be enabled.

Reply:
see my comment below. this is intentional.

Reply:
They are deleted by default, they should not.

Reply:
> They are deleted by default, they should not.

This is a different test.

Reply:
> [blackwinter](https://github.com/blackwinter) assigned [TobiasNx](https://github.com/TobiasNx) and unassigned [blackwinter](https://github.com/blackwinter) [6 minutes ago](https://github.com/metafacture/metafacture-fix/pull/177#event-6162988584)[](https://github.com/TobiasNx)
> @[TobiasNx](https://github.com/TobiasNx) TobiasNx assigned [blackwinter](https://github.com/blackwinter) and unassigned [TobiasNx](https://github.com/TobiasNx) [2 minutes ago](https://github.com/metafacture/metafacture-fix/pull/177#event-6163016330)

This conversation has not been resolved, has it? That's why I assigned you.

Reply:
sorry for that